spotify_tensorflow/luigi/utils.py

```diff
@@ -32,3 +32,10 @@ def is_gcs_path(path):
     # type: (str) -> bool
     """Returns True if given path is GCS path, False otherwise."""
     return path.strip().lower().startswith("gs://")
+
+
+def get_uri(target):
```

Comment:
This is a hack to dynamically check attribute to deal with both internal hades and other luigi targets.